### PR TITLE
Minimize CREW_MESON_OPTIONS (& fix datadir), minimize CREW_CMAKE_OPTIONS*

### LINKED ISSUE #4507: [gtk3] Unable to build
```
The Meson build system
Version: 0.55.3
Source dir: /usr/local/tmp/crew/gtk+-3.24.23.tar.xz.dir/gtk+-3.24.23
Build dir: /usr/local/tmp/crew/gtk+-3.24.23.tar.xz.dir/gtk+-3.24.23/build
Build type: native build
WARNING: Unknown options: "LIB_INSTALL_DIR, SYSCONFDIR"
The value of new options can be set with:
meson setup <builddir> --reconfigure -Dnew_option=new_value ...
Project name: gtk+-3.0
Project version: 3.24.23
Using 'CFLAGS' from environment with value: '-fuse-ld=lld'

meson.build:1:0: ERROR: Unable to determine dynamic linker

A full log can be found at /usr/local/tmp/crew/gtk+-3.24.23.tar.xz.dir/gtk+-3.24.23/build/meson-logs/meson-log.txt
gtk3 failed to build: `meson -Dbuildtype=release -Dbroadway_backend=true -Dgtk_doc=false -Ddemos=false -Dexamples=false -Dprefix=/usr/local -Dlibdir=/usr/local/lib -DLIB_INSTALL_DIR=/usr/local/lib -Dmandir=/usr/local/share/man -DSYSCONFDIR=/usr/local/etc -Ddatadir=/usr/local/lib -Dbuildtype=release build` exited with 1
```

### PR DESCRIPTION
**Pull request by satmandu**
Fixes #4507 #4508  (once they are set to use CREW_MESON_OPTIONS)
This affects at least pango, harfbuzz, and gtk3.

Works properly:
- [x] x86_64

### CODE REVIEW

**Comment by satmandu** [comment]
@uberhacker Please merge this or a version of this before rebuilding any meson dependent packages. Thanks!

File: lib/const.rb [hunk 1]
```diff
@@ -61,5 +61,5 @@
   CREW_BUILD = 'x86_64-cros-linux-gnu'
 end
 CREW_OPTIONS = "--prefix=#{CREW_PREFIX} --libdir=#{CREW_LIB_PREFIX} --mandir=#{CREW_MAN_PREFIX} --build=#{CREW_BUILD} --host=#{CREW_BUILD} --target=#{CREW_BUILD}"
-CREW_MESON_OPTIONS = "-Dprefix=#{CREW_PREFIX} -Dlibdir=#{CREW_LIB_PREFIX} -DLIB_INSTALL_DIR=#{CREW_LIB_PREFIX} -Dmandir=#{CREW_MAN_PREFIX} -DSYSCONFDIR=#{CREW_PREFIX}/etc -Ddatadir=#{CREW_LIB_PREFIX} -Dbuildtype=release"
```
Comment (uberhacker) [inline lib/const.rb:64]: Bump crew version?  How about `CREW_CMAKE_OPTIONS`?

Comment (satmandu) [reply lib/const.rb:64]: I didn't want to touch CMAKE_OPTIONS until MESON was taken care of.

Comment (satmandu) [reply lib/const.rb:64]: But yes I want to clean CREW_CMAKE_OPTIONS up  too by taking out ```-DSYSCONFDIR=#{CREW_PREFIX}/etc -Ddatadir=#{CREW_LIB_PREFIX}```

Comment (satmandu) [reply lib/const.rb:64]: The question remaining is: Does ```--build=#{CREW_BUILD} --host=#{CREW_BUILD} --target=#{CREW_BUILD}"``` work with cmake?

**Comment by satmandu** [comment]
@uberhacker I split ```#{CREW_CMAKE_OPTIONS}``` into ```#{CREW_CMAKE_OPTIONS}``` & ```#{CREW_CMAKE_LIBSUFFIX_OPTIONS}``` since some packages require ```-DLIB_SUFFIX```, and some don't.

**Comment by uberhacker** [comment]
We're probably making this more complicated than necessary.  Why not just make one `CREW_MESON_OPTIONS` that includes the `-DLIB_SUFFIX`?

**Comment by satmandu** [comment]
@uberhacker ```-DLIB_SUFFIX``` is for cmake. 

The problem is that if we have DLIB_SUFFIX=64 and it isn't needed there is a possibility of files being put in ```lib6464``` as per discussion at https://gitlab.kitware.com/cmake/cmake/-/issues/18640#note_489276

**Comment by satmandu** [comment]
If it isn't used you get a situation like this:

```
-- Found WAYLAND_CLIENT: /usr/local/lib64/libwayland-client.so  
-- Found WAYLAND_SERVER: /usr/local/lib64/libwayland-server.so  
-- Found WAYLAND_EGL: /usr/local/lib64/libwayland-egl.so  
-- Found WAYLAND_CURSOR: /usr/local/lib64/libwayland-cursor.so  
-- Found WAYLAND: /usr/local/lib64/libwayland-client.so;/usr/local/lib64/libwayland-server.so;/usr/local/lib64/libwayland-egl.so;/usr/local/lib64/libwayland-cursor.so  
-- Looking for secure_getenv
-- Looking for secure_getenv - found
-- Looking for __secure_getenv
-- Looking for __secure_getenv - not found
-- The ASM compiler identification is Clang
-- Found assembler: /usr/local/bin/clang
-- Looking for cet.h
-- Looking for cet.h - not found
-- Configuring done
-- Generating done
CMake Warning:
  Manually-specified variables were not used by the project:

    LIB_SUFFIX
```

**Pull request merged**